scripts/machine-launch-script.sh

```diff
@@ -212,108 +212,15 @@ set -o pipefail
-    #       |                                                           ~~~~~^~~~
-    # In file included from ./../libasm/libasm.h:35,
-    #                  from ./libeblP.h:33,
-    #                  from ebl_syscall_abi.c:33:
-    # ./libebl.h:254:46: note: previously declared as an array 'int[6]'
-    #   254 |                             int *callno, int args[6]);
-    #       |                                          ~~~~^~~~~~~
-    #
-    # pin to gcc=10 until we get that fixed.
-
-    CONDA_PACKAGE_SPECS+=( gcc=10 gxx=10 binutils conda-gcc-specs )
-
-    # if building riscv-toolchain from source, we need to use bison=3.4 until we have
-    # https://github.com/riscv-collab/riscv-binutils-gdb/commit/314ec7aeeb1b2e68f0d8fb9990f2335f475a6e33
-    CONDA_PACKAGE_SPECS+=( bison=3.4 )
-
-    # poky deps
-    CONDA_PACKAGE_SPECS+=( python=3.8 patch texinfo subversion chrpath git wget )
-    # qemu deps
-    CONDA_PACKAGE_SPECS+=( gtk3 glib pkg-config bison flex )
-    # qemu also requires being built against kernel-headers >= 2.6.38 because of it's use of
-    # MADV_NOHUGEPAGE in a call to madvise.  See:
-    # https://man7.org/linux/man-pages/man2/madvise.2.html
-    # https://conda-forge.org/docs/maintainer/knowledge_base.html#using-centos-7
-    # obvi this would need to be made linux-specific if we supported other MacOS or Windows
-    CONDA_PACKAGE_SPECS+=( "kernel-headers_linux-64>=2.6.38" )
-    # firemarshal deps
-    CONDA_PACKAGE_SPECS+=( rsync psutil doit=0.35.0 gitpython humanfriendly e2fsprogs ctags bison flex expat )
-    # cross-compile glibc 2.28+ deps
-    # current version of buildroot won't build with make 4.3 https://github.com/firesim/FireMarshal/issues/236
-    CONDA_PACKAGE_SPECS+=( make!=4.3 )
-    # build-libelf wants autoconf
-    CONDA_PACKAGE_SPECS+=( autoconf automake libtool )
-    # other misc deps
+    # minimal specs to allow cloning of firesim repo and access to the manager
     CONDA_PACKAGE_SPECS+=(
         bash-completion \
-        sbt \
         ca-certificates \
         mosh \
-        gmp \
-        mpfr \
-        mpc \
-        zlib \
         vim \
         git  \
-        openjdk \
-        gengetopt \
-        libffi \
-        expat \
-        libusb1 \
-        ncurses \
-        cmake \
-        graphviz \
-        expect \
-        dtc \
-        verilator==4.034 \
         screen \
-	elfutils \
-	libdwarf-dev==0.0.0.20190110_28_ga81397fc4 \
-    )
-    # clang-format for driver coding style enforcement.
-    CONDA_PACKAGE_SPECS+=( clang-format clang-tools )
-
-    # python packages
-    # While it is possible to install using pip after creating the
-    # conda environment, pip's dependency resolution can conflict with
-    # conda and create broken environments.  It's best to use the conda
-    # packages so that the environment is consistent
-    CONDA_PACKAGE_SPECS+=( \
-        boto3==1.20.21 \
-        colorama==0.4.3 \
         argcomplete==1.12.3 \
```

Comment:
I left bash-completion and argcomplete in machine-launch-script.sh because I didn't want to think through how to handle them being moved into build-setup.sh.  It may be that they should or need to move or not move for proper function.